Core/scripts/dirac-install.py

```diff
@@ -666,26 +666,34 @@ def __loadObjectDefaults(self, rootPath, objectName):
     self.loadedCfgs.append(basePath)
     return S_OK(self.globalDefaults.getChild(basePath))
 
-  def loadInstallationLocalDefaults(self, fileName):
+  def loadInstallationLocalDefaults(self, args):
     """
     Load the configuration file from a file
 
-    :param str fileName: the configuration file name
+    :param str args: the arguments in which to look for configuration filenames
     """
-    try:
-      fd = open(fileName, "r")
-      # TODO: Merge with installation CFG
-      cfg = ReleaseConfig.CFG().parse(fd.read())
-      fd.close()
-    except Exception as excp:
-      return S_ERROR("Could not load %s: %s" % (fileName, excp))
-    self.globalDefaults.update("Installations/%s" % self.instName, cfg)
-    self.globalDefaults.update("Projects/%s" % self.instName, cfg)
-    if self.projectName:
-      # we have an extension and have a local cfg file
-      self.globalDefaults.update("Projects/%s" % self.projectName, cfg)
+    # at the end we load the local configuration and merge with the global cfg
+    for arg in args:
+      if len(arg) > 4 and arg.find(".cfg") == len(arg) - 4 and ':::' not in arg:
+        fileName = arg
 
-    return S_OK()
+      logNOTICE("Defaults for LocalInstallation are in %s" % fileName)
+      try:
+        fd = open(fileName, "r")
```

Comment:
Am I wrong? `fileName` might be not defined at this point, and the exception treatment would raise `NameError`.

Reply:
You're right but it wasn't intended to be able to get this far. It's fixed in 51d0e657b906b68b8c31990680b2b7491255bec6